engine/access/subscription/streamer.go

```diff
@@ -83,10 +93,13 @@ func (s *Streamer) Stream(ctx context.Context) {
 				s.sub.Close()
 				return
 			}
+
+			// TODO: Can we change it to return no error when a client disconnects (it is done in a new subscription PR)
```

Comment:
I think this makes sense, we would just need to make sure that the behavior is clearly documented and all current implementations handle it correctly.